base_multi_image/hooks.py

```diff
@@ -76,9 +76,11 @@ def uninstall_hook_for_submodules(cr, registry, model):
         Model technical name, like "res.partner". All multi-images for that
         model will be deleted
     """
-    Image = registry["base_multi_image.image"]
-    images = Image.search([("owner_model", "=", model)])
-    images.unlink()
+    env = api.Environment(cr, SUPERUSER_ID, dict())
```

Comment:
Admittedly, laziness. I copied this from the env creation above instead of typing the ever so many characters 😉 

Good nitpick though, `dict()` is totally not pythonic - and actually I think slightly less efficient due to method call.